sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracer.java

```diff
@@ -79,4 +78,13 @@ public SpanBuilder spanBuilder(String spanName) {
   InstrumentationScopeInfo getInstrumentationScopeInfo() {
     return instrumentationScopeInfo;
   }
+
+  // Visible for testing
+  boolean isEnabled() {
+    return tracerEnabled;
+  }
+
+  void updateTracerConfig(TracerConfig tracerConfig) {
```

Comment:
Removed a comment on this method which implied it may one day be public, since this class is package private and will stay that way.